tests/addresslib/address_test.py

```diff
@@ -138,7 +138,7 @@ def test_edge_cases():
 def test_display_name__to_full_spec():
     eq_('"foo (\\"bar\\") blah" <foo@bar.com>',
         EmailAddress('foo ("bar") blah', 'foo@bar.com').full_spec())
-    eq_('"foo. bar" <foo@bar.com>',
+    eq_('foo. bar <foo@bar.com>',
```

Comment:
I would say that we should not produce incorrect output just because some ESP has a bug that adds quotes and they are not supposed to.